### ISSUE TITLE
BUG: 2.4.0 Some settings do not appear to save

### ISSUE THREAD

**Issue by rsbx** [trevorsandy/lpub3d#457]
When attempting to test the 2.4.0 release, some settings can be changed but do not get saved while others do.

Environment:
2.4.0 on Debian 10 (from the OBS Debian 10 package) using the native renderer.

Specific settings that do not save that I found are (so far):

Ldraw path - it's tedious to have to set it every time  I start lpub3d24.

Antialiasing - it defaults to 8 and can be changed but after the mandatory restart it's set to 8 again. Setting it to 0 in the config file has no effect and seems to be ignored. This is keeping me from testing since with antialiasing enabled, no images are created (a existing issue, even with LeoCAD, in the VMs on my system. With anitaliasing off, LeoCAD works fine.

**Comment by trevorsandy** [comment]
Thank you for reporting this behaviour. I’ll take a look. 

The native renderer uses some legacy OpenGL calls that may not be properly supported - particularly when running in a VM. I have seen similar behaviour with my macOS development which which is performed in a VM. 

Not being able to save settings should not be happening however.

Cheers,

**Comment by trevorsandy** [comment]
> Antialiasing - it defaults to 8 and can be changed but after the mandatory restart it's set to 8 again.

I have changed the 3DViewer AA setting from being overwritten by the LPub3D specified default on startup. This behaviour was not intended. I also made the same change to the build modification setting but this flag is dynamically reset on file load so there should not have been any adverse behaviour before this change.

> Ldraw path - it's tedious to have to set it every time I start lpub3d24.

I did not find any anomaly in the set LDraw library path routine. What steps are you taking, and where are you attempting, to set this path ? Via the LPub3D Preferences dialogue, are you using the browse button or directly entering your path into the line edit dialogue ?

As you know, it is not possible to set any folder paths from the 3DViewer Preferences dialogue - the browse buttons are hidden and line edit dialogues are read-only.

Cheers,

**Comment by rsbx** [comment]
Antialiasing setting:
The VM has some issues when it comes to 3D but LeoCad will work fine (but slow) there if I turn antialiasing off. So I attempting the same workaround for lpub3d24 but couldn't get past the forced setting. I'll try testing again when there's an updated Debian 10 package set.

LDraw path:
My apologies! I **am** able to set the LDraw library path in a remembered fashion on the VM where I actually installed the package. I was not able to set the LDraw library path on the host where I did **not** install the package but instead copied over just the lpub3d24 executable to /tmp to see if I could get past the antialiasing setting issue.
